scripts/upgrade

```diff
@@ -114,7 +109,7 @@ UPDATE SidebarCategories set sorting = 'manual' where sorting='';
 UPDATE SidebarCategories set sorting = 'manual' where sorting IS NULL;
 EOM
 
-    ynh_psql_execute_as_root --sql="$fix_old_version_sql" --database=$db_name
+    ynh_psql_db_shell <<< "$fix_old_version_sql"
```

Comment:
Will this properly execute as DB root - and not as the $app user?

Reply:
yes